src/main/java/com/todoist/pojo/Utils.java

```diff
@@ -8,4 +13,26 @@ static boolean equals(Object a, Object b) {
     static int clamp(int val, int min, int max) {
         return val < min ? min : (val > max ? max : val);
     }
+
+    public static <T> Set<T> unmodifiableSet(Collection<? extends T> s) {
+        if (s != null) {
+            return Collections.unmodifiableSet(new HashSet<T>(s));
+        } else {
+            return Collections.emptySet();
```

Comment:
The result should be the same, because the empty set is immutable (will just crash) and it's a constant in `Collections`. But we can do it for obviousness if you want to!